src/components/qas/questions/qaCalculator/qaCalculatorUtilities.js

```diff
@@ -64,5 +121,129 @@ export const calculateCraterDiameter = ({
+  return craterDiameter / (2 * Math.sqrt(2));
+};
+
+export const getMercalliIntensity = richterMagnitude => {
+  const mercalliIntensity = [
+    'No damage felt.',
+    '<em style="font-weight: 500;">I.</em> Not felt except by a very few under especially favorable conditions.',
+    '<em style="font-weight: 500;">II.</em> Felt only by a few persons at rest, especially on upper floors of buildings.',
+    '<em style="font-weight: 500;">III.</em> Felt quite noticeably by persons indoors, especially on upper floors of buildings. Many people do not recognize it as an earthquake. Standing motor cars may rock slightly. Vibrations similar to the passing of a truck.',
+    '<em style="font-weight: 500;">IV.</em>Felt indoors by many, outdoors by few during the day. At night, some awakened. Dishes, windows, doors disturbed; walls make cracking sound. Sensation like heavy truck striking building. Standing motor cars rocked noticeably.',
+    '<em style="font-weight: 500;">V.</em> Felt by nearly everyone; many awakened. Some dishes, windows broken. Unstable objects overturned. Pendulum clocks may stop.',
+    '<em style="font-weight: 500;">VI.</em> Felt by all, many frightened. Some heavy furniture moved; a few instances of fallen plaster. Damage slight.',
+    '<em style="font-weight: 500;">VII.</em> Damage negligible in buildings of good design and construction; slight to moderate in well-built ordinary structures; considerable damage in poorly built or badly designed structures; some chimneys broken.',
+    '<em style="font-weight: 500;">VIII.</em> Damage slight in specially designed structures; considerable damage in ordinary substantial buildings with partial collapse. Damage great in poorly built structures. Fall of chimneys, factory stacks, columns, monuments, and walls. Heavy furniture overturned.',
+    '<em style="font-weight: 500;">IX.</em> General panic. Damage considerable in specially designed structures; well-designed frame structures thrown out of plumb. Damage great in substantial buildings, with partial collapse. Buildings shifted off foundations. Serious damage to reservoirs. Underground pipes broken. Conspicuous cracks in ground. In alluviated areas sand and mud ejected, earthquake fountains, sand craters.',
+    '<em style="font-weight: 500;">X.</em> Most masonry and frame structures destroyed with their foundations. Some well-built wooden structures and bridges destroyed. Serious damage to dams, dikes, and embankments. Large landslides. Water thrown on banks of canals, rivers, lakes, etc. Sand and mud shifted horizontally on beaches and flat land. Rails bent slightly.',
+    '<em style="font-weight: 500;">XI.</em> As X. Rails bent greatly. Underground pipelines completely out of service.',
+    '<em style="font-weight: 500;">XII.</em> As X. Damage nearly total. Large rock masses displaced. Lines of sight and level distorted. Objects thrown into the air.',
+  ];
+
+  return mercalliIntensity[Math.round(richterMagnitude)];
+};
```

Comment:
It's usually bad form to throw blocks of stringified html into an array like this.  I'd like to see this take a format more akin to the Air Blast damage descriptions.  Furthermore, I think that's pretty doable as I don't think the Mercalli Intensities are useful here.  A lot/everybody knows about the richter scale, so I thought it would be fun to include that number, but no one has ever heard of this other scale.  If you aren't having to bold the roman numerals are you then able to treat these descriptions the same way you did the air blast ones?

"As X." won't mean anything for the user since we are not (by design currently) giving them access to the with the full table of descriptions, so let's go ahead and replace both of those with the full description of "X": "Most masonry and frame structures destroyed with their foundations. Some well-built wooden structures and bridges destroyed. Serious damage to dams, dikes, and embankments. Large landslides. Water thrown on banks of canals, rivers, lakes, etc. Sand and mud shifted horizontally on beaches and flat land. Rails bent slightly" (and should be tweaked where appropriate to avoid redundant language).

And one more note here: I think your approach here (clever rounding + indexing + look up of the richter value) is probably why richter values are not lining up as they are in the the table (as notes in the main review comment).  I believe this approach would work if the descriptions lined up 1:1 with richter values, but there are jumps in terms of what ranges correspond to which ranges of intensities, so I think you are going to have to use an approach similar to what you're doing with the Air Blast look up.  Otherwise I do agree that if a richter rounds up or down only show that one corresponding mercalli description.